docs/src/pages/components/lists/lists.md

```diff
@@ -47,7 +47,7 @@ Below is an interactive demo that lets you explore the visual results of the dif
 
 ## Align list items
 
-You should change the list item alignment when displaying 3 lines or more, set the `alignItems` prop to "flex-start".
+To change the list item alignment when displaying 3 lines or more, set the `alignItems` prop to "flex-start".
```

Comment:
This changes the semantic otherwise

```suggestion
You should change the list item alignment when displaying 3 lines or more.
You can set the `alignItems="flex-start"` to do so.
```